core/src/test/java/io/narayana/nta/test/CentralisedAS8IntegrationTest.java

```diff
@@ -76,6 +76,7 @@ public static WebArchive createDeployment() {
                 .addAsWebInfResource(new FileAsset(new File("src/test/resources/persistence.xml")),
                         "classes/META-INF/persistence.xml")
                 .addAsManifestResource(new FileAsset(new File("src/test/resources/nta-test-ds.xml")), "nta-test-ds.xml")
+	        .addAsManifestResource(new FileAsset(new File("src/test/resources/filter.properties")), "filter.properties") 
```

Comment:
I would recommend  getting resources via the class loader instead of using hard code paths.